### PR TITLE
fix(K8s): fixing what's new

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

**Review by paperclypse — DISMISSED** [review]
Is this just adding the .md file suffix?

**Comment by jeff-colucci** [comment]
Correct, this just needs the correct file tag to show up in the WN nav I believe. It didn't have one in the original PR, but the weird thing was it was actually showing up in the build preview so I didn't think there was any actual issue.

Either way, it should be fixed now!

**Comment by jeff-colucci** [comment]
The only change between this and the review was a fix to the image formatting so I'm going to push it out since it built successfully.

File: src/content/whats-new/2024/09/whats-new-09-11-logs-in-context-in-kubernetes-deployments.md [hunk 1]
```diff
@@ -12,17 +12,6 @@ To get started, open a Kubernetes deployment view and click the **Logs** tab. A
...
-
-<img
-    title="A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page"
-    alt="A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page"
-    src={/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp}
-/>
-<figcaption>A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page.</figcaption>
+![Logs in context view](/images/WhatsNewLogsInContextKubernetesDeployment.webp "A screenshot showing logs in context view for a kubernetes deployment")
 
+![LogsMiniOverviewWidget](/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp "A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page")]
```
Comment (paperclypse) [inline src/content/whats-new/2024/09/whats-new-09-11-logs-in-context-in-kubernetes-deployments.md:17]: @jeff-colucci There's a stray square bracket at the very end of this line that needs to be deleted, once you do that, and the builds finish, feel free to merge it in

Comment (paperclypse) [reply src/content/whats-new/2024/09/whats-new-09-11-logs-in-context-in-kubernetes-deployments.md:17]:
```suggestion
![LogsMiniOverviewWidget](/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp "A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page")
```

**Pull request merged**